docs/install.md

```diff
@@ -27,37 +27,44 @@ Yazi can be _optionally_ extended with other command line tools to enable additi
 sudo pacman -S yazi ffmpegthumbnailer unarchiver jq poppler fd ripgrep fzf zoxide
 ```
 
-If you want to use the latest git version, you can install it from [AUR](https://aur.archlinux.org/packages/yazi-git/) or [Arch Linux CN](https://github.com/archlinuxcn/repo/):
+If you want to use the latest Git version, you can install it from [AUR](https://aur.archlinux.org/packages/yazi-git/) or [Arch Linux CN](https://github.com/archlinuxcn/repo/):
 
 ```bash
 paru -S yazi-git ffmpegthumbnailer unarchiver jq poppler fd ripgrep fzf zoxide
 ```
 
-## macOS
+## macOS / Homebrew
 
-Install Yazi and its dependencies with Homebrew:
+First, make sure that Homebrew is fully up-to-date with `brew update`.
+
+Then you can install Yazi by itself:
+
+```bash
+brew install yazi
+```
+
+Or along with the optional dependencies:
 
 ```bash
-brew update
```

Comment:
https://github.com/sxyazi/yazi/issues/392 this is the reason why i add it lol

Reply:
Yeah I know... look at line 38 above this.